src/client/Microsoft.Identity.Client/Utils/DateTimeHelpers.cs

```diff
@@ -63,15 +63,39 @@ public static long GetDurationFromWindowsTimestamp(string windowsTimestampInFutu
             return (long)unixTimestamp - CurrDateTimeInUnixTimestamp();
         }
 
-        public static long GetDurationFromNowInSeconds(string unixTimestampInFuture)
+        public static long GetDurationFromNowInSeconds(string dateTimeStamp)
         {
-            if (string.IsNullOrEmpty(unixTimestampInFuture))
+            if (string.IsNullOrEmpty(dateTimeStamp))
```

Comment:
Btw, I think this function is a good example of one that can actually benefit from unit testing.

Reply:
there is a unit test for this new method